Cisco fix access list template

guillaume-mbali opened the pull request:
**ISSUE TYPE**
Fix Template Pull Request

**COMPONENT**
cisco_ios_show_access-list.textfsm, ios, show access-list

**SUMMARY**
Correction of template to match more rules

whitej6 committed:
release 2.2.2 (#967)

scetron committed:
Add Aruba AOS-CX command support (#927)

joewesch committed:
Enhancement: cisco_ios_show_ip_access-lists - Adds parsing of log tags (#970)

FragmentedPacket committed:
Release v2.3.0 (#971)

mitchell-foxworth committed:
Bugfix: cisco_ios_show_ip_access-lists - Account for single match (#973)

FragmentedPacket committed:
Release v2.3.1 (#974)

thomasbridge74 committed:
Allow cisco ios show ip bgp sum to parse when local AS has a "." in the AS number - fixes #986 (#987)

Kani999 committed:
Bugfix: cisco_nxos_show_interface_status: Allow blank Type to capture (#979) (#980)

a-finocchiaro committed:
Bugfix: cisco_ios_show_interfaces_switchport.textfsm: Account for App Interface line (#968)

FragmentedPacket committed:
Release v2.3.2 (#988)

ahlara-devcore committed:
cisco_ios_show_access-session: Adding Identity to MAC column (#990)

Co-authored-by: Alejandro Hernandez <alejandro.hernandez@toronto.ca>

georgesnow committed:
[New Template] Ciena - traffic-profile standard-profile  (#981)

- Add Cienna traffic-profile standard-profile

AJatCDW committed:
New template: cisco_nxos_show_ip_interface_vrf_all.textfsm (#978)

* New template: cisco_nxos_show_ip_interface_vrf_all.textfsm

* Update index

* Update index

antonalekseev committed:
New template: juniper_junos_show_system_uptime.textfsm (#975)

New template: juniper_junos_show_system_uptime.textfsm

QuasarKid committed:
Template Change: cisco_ios, show archive (#905)

antonalekseev committed:
Bugfix: change date format in hp_comware_display_clock.textfsm (#977)

As per
https://techhub.hpe.com/eginfolib/networking/docs/switches/5940/5200-1008b_fund_cr/content/491962772.htm
output of the `display clock` commands output contains date in `MM/DD/YYYY`
format. Can confirm this on different types of HPE switches I have at my disposal.

Added test cases that cover the scenarios when time zone and daylight time
saving configurations are set.

scetron committed:
update arista interface template + raw (#963)

lamiskin committed:
New Template: cisco_ios_show_dhcp_lease (#991)

armartirosyan committed:
Fix ios_mac-addr type2

armartirosyan committed:
Merge pull request #994 from networktocode/cisco-ios-show-mac-fix-type2

Fix ios_mac-addr type2

dainok committed:
fix parsing with int addresses = 0 (#982)

Fix parsing of int addresses = 0

ak-empiak committed:
new template : huawei_vrp dis_int_desc + dis_port_vlan ; fix huawei_vrp dis_int_br (#998)

Add new templates `huawei_vrp` `dis_int_desc`  `dis_port_vlan`

dbell-infra committed:
added VLAN value and search pattern (#1002)

* Add VLAN value for Cisco ASA `show interface`

armartirosyan committed:
Enhancements: Cisco IOS show mac and show module (#1006)

georgesnow committed:
Junos show chassis (#997)

* baseline

* baseline ciena traffic-profile

* traffic-p stand pro show parsing

* fixed filenames

* fixing naming, linting etc...

* remove stale file

* juniper extend interfaces for capturing ips

* removing commented outlines

* junos show chassis hardware template

* fix end of file

* fix end of file

* updating index file

* needed to add another match

* clean up comments

* add matching for fpc,mic,pic

* removing commented out lines for fpc,mic,pic capture

* account for devices that without mics and multiple xcvrs

* updated tests to account for additional xcvrs

* fixed template to have error out this includes the raw capture for 3 platforms need to update ymls for tests

* first set of ymls for tests

* Change interfaces to use Continue.Record

* updated ymls for tests

* removed commented out lines

* fixing raw3 tests

* moving EOF the correct location

* Fix continues and simplify template

Co-authored-by: gfn4 <gfn4@unh.edu>
Co-authored-by: jmcgill298 <jacob@networktocode.com>

jvanderaa committed:
Prep for release v3.0.0 (#1010)

Release 3.0.0 prep

yone2ks committed:
New Template: Cisco ASA: 'show cpu usage detailed' (#1014)

Adds new template - Cisco ASA: `show cpu usage detailed`

armartirosyan committed:
Fix IOS show mac address Type3 and Type4. Additional test files (#1019)

Updates show mac address type3 and Type4

zamelek committed:
Fix template IOS show int status - added new interface status (#1023)

Add suspended as choice on Cisco IOS show interfaces status.

jvanderaa committed:
Adds FTD output example from Cisco support site (#1032)

ksaegusa committed:
Add show ip nat translations template (#1028)

Add cisco ios show ip nat translations template

scetron committed:
Fixes #1043 Update arista_eos_show_interfaces_description.textfsm (#1044)

* Update arista_eos_show_interfaces_description.textfsm - Update choices for Protocol to include 'notpresent' option.

hagleyj committed:
aruba_os_show_ap_database (#1042)

Adds aruba_os_show_ap_database

shanecbauman committed:
Modified cisco_ios_show_boot (#1038)

Updates cisco_ios_show_boot

diepes committed:
Fixes_992-cisco_nxos_show_ip_bgp.textfsm (#1033)

Fixes nxos show ip bgp.

diepes committed:
fixs-1027-cisco_nxos-show_ip_interface (#1046)

Updates to cisco_nxos-show_ip_interface and cisco_ios_show_boot

lamiskin committed:
Improve cisco_nxos show_ip_route (#1093)

lamiskin committed:
Improve cisco_nxos show_ip_interface (#1092)

lamiskin committed:
Improve cisco_ios show_standby (#1091)

lamiskin committed:
Improve cisco_ios show_module (#1090)

lamiskin committed:
Improve cisco_ios show_ip_interface (#1089)

lamiskin committed:
Improve cisco_ios show_interfaces_switchport (#1088)

lamiskin committed:
Improve cisco_ios show_etherchannel_summary (#1087)

lamiskin committed:
Improve cisco_ios show_lldp_neighbors_detail (#1086)

* Improve cisco_ios show_lldp_neighbours_detail

* Ensure yml file has new line at EOF

* Include @diepes suggestions.

lamiskin committed:
Improvements to cisco_asa show_interface (#1084)

network-shark committed:
show platform diag now returns all linecards and their rommon versions (#1081)

* Fixing show_platform_diag current template does not recognize slot numbers correctly

* make tox happy

Co-authored-by: Sebastian Schaack <sharky@MacBook-Pro-von-Sebastian.local>

hagleyj committed:
fix for 1051 (#1055)

Co-authored-by: Jeff Hagley <hagleyj@umich.edu>

elavaud committed:
Add ubiquiti edgerouter show version (#1065)

jvanderaa committed:
Update line feed to LF from CRLF (#1096)

jvanderaa committed:
Clarifies docs about length order. (#1099)

Niltak committed:
fixed #1024 - Moved Record to Device ID (#1039)

Moving the record from the dashes to a continued 'Device ID'. This will fix the issue while maintaining the current output expectation.

dainok committed:
update hp procurve show_mac-address (#1108)

diepes committed:
fixes#1094-new-cisco-apic (#1112)

Co-authored-by: Pieter E Smit <diepes@github.com>

diepes committed:
fixes#1083-cisco_ios_show_module (#1117)

add cisco_ios_show_module5.raw stack of switches

dainok committed:
fix not advertised vlan (#1114)

fix not advertised vlan

mjuenema committed:
Template for Cisco IOS show ip access-lists now parse network and service object groups. All tests have been updated accordingly. (#1113)

verbosemode committed:
Add Cisco IOS XR show install active (#1105)

dainok committed:
add vlan_id to show interfaces (#1115)

* add vlan_id to show interfaces

* track also QinQ vlans

* rename vlan inner/outer fields

diepes committed:
fixes#999-cisco_nxos_show_int_brief-mgmt (#1119)

ajschroeder committed:
added cisco_ios_show_ip_http_server_status.textfsm

pauljorgenson committed:
Add arista show processes top once (#1110)

Arista show processes top once

jcoeder committed:
Added FPM to the Start and XCVR States

showipintbri committed:
Added Juniper Junos `show vlans` (#1125)

* Added index and textfsm files for Juniper 'show vlans'

networkop committed:
fix EOS bug

itdependsnetworks committed:
Merge pull request #1127 from networkop/master

fix EOS 4.28.XF bug

hagleyj committed:
Add aruba_os_show_ap_radio-database (#1133)

* aruba_os_show_ap_radio-database

Elinpf committed:
fix: Path separator error on windows (#1139)

elavaud committed:
Add zyxel cfg nat get (#1073)

* Add zyxel cfg nat get

* Update ntc_templates/templates/zyxel_os_cfg_nat_get.textfsm

Co-authored-by: Ken Celenza <ken@celenza.org>

Co-authored-by: Edouard Lavaud <e.lavaud@unyc.io>
Co-authored-by: Ken Celenza <ken@celenza.org>

jvanderaa committed:
add new template ubiquiti_edgerouter_show_dhcp_leases (#1097)

Co-authored-by: Edouard Lavaud <e.lavaud@unyc.io>

hagleyj committed:
Create aruba_os_show_ap_database_long (#1134)

* fix for 1051

* aruba_os_show_ap_radio-database

* aruba_os_show_ap_database_long

Co-authored-by: Jeff Hagley <hagleyj@umich.edu>

elavaud committed:
Add mikrotik templates: (#1136)

* interface ethernet monitor (name) once
* ip arp print without-paging
* ip firewall filter print all without-paging
* ip dhcp-server lease print without paging
* ip firewall nat print all without-paging
* snmp community print without-paging
* ip route print without-paging
* system routerboard print

Co-authored-by: Edouard Lavaud <e.lavaud@unyc.io>

diepes committed:
fixs#1103-cisco_show_boot (#1111)

add new value BOOT_OPTIMIZATION
fix capture of CONFIG_DOWNLOAD_DHCP_NEXT_BOOT

Co-authored-by: Pieter E Smit <diepes@github.com>

diepes committed:
fix#826-2 cisco_ios_show_ip_eigrp_topology.textfsm add ADV_FD and ADV_RD (#1013)

viktorkertesz committed:
fix cisco_ios show ip bgp output where metric is wider than 6 chars (#1123)

* fixed metric width when it is wider than 6 chars

* fix linting

* new line added

Co-authored-by: Jacob McGill <9847006+jmcgill298@users.noreply.github.com>

Co-authored-by: Viktor Kertesz <vkertesz2@gmail.com>
Co-authored-by: Jacob McGill <9847006+jmcgill298@users.noreply.github.com>

viktorkertesz committed:
fix cisco_ios show ip bgp neighbors advertised-routes output where metric is wider than 6 chars (#1124)

* fixed metric width when it is wider than 6 chars

* fix linting

* new line  added

Co-authored-by: Jacob McGill <9847006+jmcgill298@users.noreply.github.com>

Co-authored-by: Viktor Kertesz <vkertesz2@gmail.com>
Co-authored-by: Jacob McGill <9847006+jmcgill298@users.noreply.github.com>

Elinpf committed:
fix: change huawei_vrf shortest command (#1141)

elavaud committed:
Add zyxel cfg ipalias get (#1068)

Co-authored-by: Edouard Lavaud <e.lavaud@unyc.io>

elavaud committed:
Add new template for ubiquiti edgerouter show interfaces ethernet physical (#1064)

Co-authored-by: Edouard Lavaud <e.lavaud@unyc.io>

elavaud committed:
Add multiple zyxel templates (#1142)

* add zyxel_os_sys_atsh

* Add Zyxel OS cfg_firewall_acl_get

* Add Zyxel cfg intf group get

* Add Zyxel cfg lan get --Name name

* Add Zyxel cfg lan get

* Add Zyxel cfg lanhosts get

* Add Zyxel cfg nat addr map get

* Add Zyxel cfg snmp get

* Add Zyxel cfg static dhcp get

* Add Zyxel cfg static_route get

* Add Zyxel cfg wlan get

* Add zyxel zycli Ethctl media-type

Co-authored-by: Edouard Lavaud <e.lavaud@unyc.io>

elavaud committed:
add new template ubiquiti_edgerouter_show_arp (#1062)

Co-authored-by: Edouard Lavaud <e.lavaud@unyc.io>

k-ribot committed:
Add ubiquiti edgerouter show ipv6 neighbors

dainok committed:
add Cisco XR show ipv4 int (#1162)

Add Cisco XR show ipv4 int

ChristopherJHart committed:
Add Cisco NX-OS "show ip dhcp snooping statistics" template (#1177)

Add Cisco NX-OS "show ip dhcp snooping statistics" template

nsnelson402 committed:
Eigrp interfaces detail (#1181)

* Added new template and tests for EIGRP Interfaces Detail

ReK42 committed:
Updated support for Arista EOS devices (#1174)

Arista updates:

* Update `show ip bgp summary` to support description column
* Update `show ip ospf neighbor` to support instance column
* Update `show vrf` to support line card interfaces, subinterfaces and an alternate header
* Add support for `show ip ospf interface brief`
* Add support for `show pim ipv4 interface`
* Add support for `show pim ipv4 neighbor`
* Add support for `show ip ospf summary`

nsnelson402 committed:
Cisco ios show snmp group (#1183)

* Added new template and tests.

* Created cisco_ios_show_snmp_group.textfsm template.

Co-authored-by: Nicholas Nelson - 10105570 <nicholas.nelson@fiserv.com>

whitej6 committed:
v3.1.0 release prep (#1185)

dependabot[bot] committed:
Bump pyyaml from 5.3.1 to 5.4 (#1186)

Bumps [pyyaml](https://github.com/yaml/pyyaml) from 5.3.1 to 5.4.
- [Release notes](https://github.com/yaml/pyyaml/releases)
- [Changelog](https://github.com/yaml/pyyaml/blob/master/CHANGES)
- [Commits](https://github.com/yaml/pyyaml/compare/5.3.1...5.4)

---
updated-dependencies:
- dependency-name: pyyaml
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

caballero-oscuro committed:
Update cisco_ios_show_port-security_interface_interface.textfsm

Value LAST_SRC_MAC_ADDR_VLAN (\w.+:\d+)

Change regex for mac address to be any alphanumeric character instead of any numeric digit.

nsnelson402 committed:
Cisco ios show crypto pki certificates (#1187)

* Added new template and tests.

* Fixed changelog

Co-authored-by: Nicholas Nelson - 10105570 <nicholas.nelson@fiserv.com>

joewesch committed:
Migrates CI from Travis to GitHub Actions (#1192)

* Migrates CI from Travis to GitHub Actions

* Removes python 3.6 from testing matrix

nsnelson402 committed:
Cisco ios show crypto ipsec sa detail (#1191)

* Added new template and tests for show crypto ipsec sa detail

dainok committed:
add linux templates (#1193)

Add linux templates

elavaud committed:
fix(mikrotik): system routerboard print - fix corner case where warning messages are added (#1196)

Fix `system routerboard print` corner case.

elavaud committed:
Mirkotik: Fix issue with flags on IP route print and truncated long IPv6 (#1152)

Fix flags on ip route print mikrotik

k-ribot committed:
Fix Zyxel sys atsh template when there message before data (#1197)

k-ribot committed:
Add Huawei VRP display routing-table template

k-ribot committed:
MikroTik routeros exclude all log messages in templates

k-ribot committed:
MikroTik routeros add comment field to route print template (#1210)

k-ribot committed:
Update Ubiquiti Edgerouter show dhcp leases template (#1209)

elavaud committed:
New template: Huawei - display arp brief (#1208)

Co-authored-by: Edouard Lavaud <e.lavaud@unyc.io>

charlesmonson committed:
Fix cisco_xr_show_arp with Management/RP interface present (#1207)

* Fix show arp template to allow presense of 'RP' in CPU when Mgmt interface is in use

* Fix show arp template to allow presense of 'RP' in CPU when Mgmt interface is in use

cuong-nguyenduy committed:
Fix Huawei VRP `display version`template (#1205)

* Fix Huawei VRP `display version`

cuong-nguyenduy committed:
Fix Cisco NXOS `display inventory` (#1206)

Reordering the order of Regex such that the more specific will be used before the more generic ones

k-ribot committed:
Update Huawei VRP display interface brief template (#1201)

k-ribot committed:
Add Huawei VRP display ipv6 neighbors template (#1212)

k-ribot committed:
Add Huawei VRP display startup template (#1211)

k-ribot committed:
Update Huawei VRP display version template (#1200)

k-ribot committed:
Add Huawei VRP display sn license template (#1199)

itdependsnetworks committed:
Merge pull request #1144 from unyc-io/ubiquiti_edgerouter_show_ipv6_neighbors

Add ubiquiti edgerouter show ipv6 neighbors

elavaud committed:
Huawei VRP: Add 2 new templates for display service-set (#1213)

Adds templates for display service-set

k-ribot committed:
Add Huawei VRP display snmp-agent community template (#1216)

itdependsnetworks committed:
Change docs to mkdocs, update dev environment, move to click for scripts

k-ribot committed:
Add Huawei VRP display nat server template (#1219)

k-ribot committed:
Add Huawei VRP display acl and acl ipv6 template (#1218)

itdependsnetworks committed:
Merge pull request #1217 from itdependsnetworks/redo-docs

Change docs to mkdocs, update dev environment, move to click for scripts

itdependsnetworks committed:
Update icon image and ci file

itdependsnetworks committed:
Merge pull request #1221 from itdependsnetworks/ci-icon

Update icon image and ci file

k-ribot committed:
Add mikrotik routeros ipv6 neighbor print template (#1146)

k-ribot committed:
Add ubiquiti edgerouter show dhcpv6 server leases (#1145)

elavaud committed:
Huawei - new template - display traffic-filter applied-record (#1223)

ryanmerolle committed:
address bug 1225 (#1226)

Update arista_eos_show_ip_route for message of no ip routing

guillaume-mbali committed:
Add Cisco ipv6 route template (#1220)

ReK42 committed:
Updated support for Cisco IOS-XR devices (#1176)

* Add support for `show version brief`
* Add support for `show platform summary location <location>`
* Add support for `show vrf all detail`
* Add support for `show ipv4 vrf <vrf> interface brief`
* Add support for `show ospf vrf <vrf> neighbor`
* Add support for `show ospf vrf <vrf> interface brief`
* Add support for `show bgp instance <instance> summary`
* Add support for `show pim ipv4 group-map`
* Add support for `show pim ipv4 neighbor`
* Add support for `show pim ipv4 interface`

jmcgill298 committed:
New Template: ruckus_fastiron, show mac-address (#896) (#1233)

Co-authored-by: Adam Troy <adam.troy@swbell.net>

itdependsnetworks committed:
added arista ipv6 bgp sum template, fixed bug in ip bgp sum template as well (#1232)

Co-authored-by: joshs <jsakofsky@netflix.com>

itdependsnetworks committed:
Merge pull request #1214 from unyc-io/huawei-vrp-add-display-routing-table-template

Add Huawei VRP display routing-table template

itdependsnetworks committed:
Merge pull request #1215 from unyc-io/mikrotik-routeros-exclude-log-messages

MikroTik routeros exclude all log messages in templates

itdependsnetworks committed:
Update icon, fix issue with missing page, make support model more clear (#1235)

k-ribot committed:
Add Huawei VRP display interface template (#1202)

Commit:
Add aruba_os show ap bss-table details (#1147)

* fix for 1051

* aruba_os_show_ap_radio-database

* aruba_os_show_ap_database_long

* aruba_os show ap bss-table details

guillaume-mbali committed:
fix(cisco): ipv6 neighbors template (#1230)

itdependsnetworks committed:
Minor doc update (#1244)

* Minor doc update

itdependsnetworks committed:
Create xr show controllers hundred gig all, add test files for show c… (#1245)

jmcgill298 committed:
Modified arista_eos_show_vlan to list Ports issue #908 (#1248)

Co-authored-by: bravindranath@aristanetworks.com <bravindranath@arista.com>

jmcgill298 committed:
Armartirosyan/cisco ios show ip bgp neighbors (#1250)

* fix for #914

* Add test data

Co-authored-by: Armen Martirosyan <armartirosyan@gmail.com>

jmcgill298 committed:
Add ipinfusion_ocnos_show_lldp_table template (#1251)

Co-authored-by: mheniMerz <mheni.merzouki@nist.gov>

sacharv committed:
juniper: add SRX and XE support to show version (#1053)

* juniper: add SRX and XE support to show version

* Apply suggestions from code review

Co-authored-by: Jacob McGill <9847006+jmcgill298@users.noreply.github.com>

diepes committed:
Fixs_1009_fortinet_get_system_ha_status.textfsm (#1041)

* Fixs_1009_fortinet_get_system_ha_status.textfsm

* Update tests/fortinet/get_system_ha_status/fortinet_get_system_ha_status_6.4.raw

Co-authored-by: Pieter E Smit <diepes@github.com>
Co-authored-by: Jacob McGill <9847006+jmcgill298@users.noreply.github.com>

diepes committed:
fix1037_paloalto_panos_show_running_nat-policy (#1040)

* fix1037_paloalto_panos_show_running_nat-policy
- add 2 more raw examples

* Update ntc_templates/templates/paloalto_panos_show_running_nat-policy.textfsm

Co-authored-by: Pieter E Smit <diepes@github.com>
Co-authored-by: Jacob McGill <9847006+jmcgill298@users.noreply.github.com>

itdependsnetworks committed:
Update doc link (#1256)

elavaud committed:
fix(Mikrotik): Optional space before end of line (#1159)

Co-authored-by: Edouard Lavaud <e.lavaud@unyc.io>

afly007 committed:
Fixed TYPE regex to include non-dynamic addresses (#1253)

* Fixed TYPE regex to include non-dynmaic addresses

* Updated TYPE regex to include non-dynmaic addresses

* Added tests for port-access-security  TYPE regex

jmcgill298 committed:
Add IPv6 to linux ip address show (#1258)

jmcgill298 committed:
Add template for IOS show ap summary (#1257)

Co-authored-by: Rishabh4290 <91332259+Rishabh4290@users.noreply.github.com>

joewesch committed:
Adds individual interface parsing (#1254)

itdependsnetworks committed:
Modify to be clear only 1 template update per PR (#1260)

jvanderaa committed:
additional templates for ruckus fastiron (#1261)

Update Ruckus Fastiron

Co-authored-by: scetron <stephen.corry@networktocode.com>

itdependsnetworks committed:
Prep 3.2.0 (#1262)

* Prep 3.2.0

* fix link

itdependsnetworks committed:
Add support for Ericsson IPOS router (#1265)

Co-authored-by: lutfisan <m_lutfi_santoso@yahoo.co.id>

itdependsnetworks committed:
add display_arp_all and display_isis_peer (#1266)

* add display_arp_all and display_isis_peer

* Fix file names and order

Co-authored-by: lutfisan <m_lutfi_santoso@yahoo.co.id>

itdependsnetworks committed:
Add support DLink DS Show ARP (#1267)

* Add support DLink DS Show ARP

* Updates to dlink

* updates

Co-authored-by: Carlos Latorre <c.latorre@civired.com>

itdependsnetworks committed:
Add support DLink DS Show ARP (#1267) (#1268)

Fix for issue #1034

Template update to fix compatibility issues with show interface status on interfaces that are undergoing cable-diagnostics tdr testing.

update

Co-Authored-By: Carlos Latorre <c.latorre@civired.com>

Co-authored-by: Carlos Latorre <c.latorre@civired.com>

elavaud committed:
Zyxel fix sys atsh parsing (#1151)

Co-authored-by: Edouard Lavaud <e.lavaud@unyc.io>

jvanderaa committed:
Fix Cisco NXOS 5K: show forwarding ipv4 route (#1263)

authored-by: Tymofii Dmytrenko

dainok committed:
Hp comware displ vpn inst det (#1259)

* implement hp comware displ vpn detail

* add error management

yucai100 committed:
Update hp_comware_display_device_manuinfo.textfsm (#1132)

Update hp_comware_display_device_manuinfo.textfsm

itdependsnetworks committed:
fix: huawei_vrf_display_version (#1269)

Co-authored-by: Elin <365433079@qq.com>

itdependsnetworks committed:
Fix Mikrotik interface ethernet monitor name once when no link partner advertising (#1270)

Co-authored-by: Edouard Lavaud <e.lavaud@unyc.io>

dainok committed:
Add HPE templates (#1246)

klausmark committed:
Add support for fortigate 7.0 (#1227)

dainok committed:
Fix case sensitivity for HP Comware (#1272)

dainok committed:
HP Comware fix spacing allowances

Fix templates if output contain tabs.

pszulczewski committed:
Fix cisco_xr_show_version.template (#1276)

Co-authored-by: Anubha Sewlani <anbalani@microsoft.com>

pszulczewski committed:
Added input and output pps for cisco_ios_show_interfaces (#1277)

Co-authored-by: Lamoni Finlayson <lamonif@gmail.com>
Co-authored-by: patryk <patryk.szulczewski@networktocode.com>

pszulczewski committed:
Merge pull request #1278 from networktocode/pr_1120

added cisco_ios_show_ip_http_server_status.textfsm

pszulczewski committed:
Merge pull request #1280 from networktocode/pr_1121

Fix to juniper_junos_show_chassis_hardware.
Added FPM to the Start and XCVR States.

mycol committed:
Added RUNTS, GIANTS, FRAME, and OVERRUN (#1284)

guillaume-mbali committed:
Ubiquiti: Add show ipv6 route template (#1274)

KeithETruesdell committed:
Add template for cisco ios ping (#1282)

guillaume-mbali committed:
Ubiquiti - add show ip route template (#1275)

sjhloco committed:
Fixed issue with timezones that have a / in them such as GMT/BST (#1285)

sjhloco committed:
Fixed issue with timezones and support for IPsecOverNatT (#1289)

sjhloco committed:
Added Process ID to parsed data (#1297)

sjhloco committed:
Added nxos show ip bgp summary address family to parsed data (#1300)

sjhloco committed:
Added support for port-channels, tunnels and a few 5585 output fixes (#1288)

KeithETruesdell committed:
cisco ios show_version capture software and release (#1292)

* added software image and release

added software image capture and the release from the initial line of the software

* update cisco ios show version to include the software image and release as additional separate fields, tests included

guillaume-mbali committed:
Corrected nexthop_ip regex in ubiquiti ipv6 route template (#1294)

sjhloco committed:
Added support for VRFs (#1296)

sjhloco committed:
added ip ospf intf br and fixed process on ospf nbr (#1298)

jversch6 committed:
Update Cisco AireOS WLC show inventory (#1286)

guillaume-mbali committed:
feat(ubiquiti): Add template ubiquiti edgerouter show interfaces (#1291)

dainok committed:
fix #1301 (#1302)

sjhloco committed:
Add new cisco asa templates (#1287)

sjhloco committed:
Add: cisco wlc templates for show flexconnect and interface group summary (#1299)

dainok committed:
Fix display interface and ip interface (#1313)

dainok committed:
NXOS show ip int fails without WCCP (#1307)

* fix template and add test

* re-trigger build

dainok committed:
Drop internet prot lines (#1305)

guillaume-mbali committed:
Cisco add template ipv6 access-lists (#1312)

k-ribot committed:
Corrected Huawei display_interface template to add Tunnel and Virtuel interface (#1311)

sjhloco committed:
Added support for IKEv2 parameters that was causing parsing to fail (#1308)

sjhloco committed:
added IOS show nve vni and nve peers templates (#1304)

sjhloco committed:
added NXOS show nve vni and nve peers templates (#1303)

elavaud committed:
Mikrotik routeros - firewall filter - Add src mac address, correct ipv6 path and empty log prefix (#1172)

Co-authored-by: Edouard Lavaud <e.lavaud@unyc.io>

jversch6 committed:
Added new template: show interface link on cisco ios (#1290)

dainok committed:
Hp comware new displ vlan lldp (#1315)

* Add display_lldp_neighbor-information_list
* Add hp_comware_display_vlan_all
* fix for hp_comware_display_interface
* one more fix for hp_comware_display_interface
* fix for cisco_ios_show_vrf

k-ribot committed:
MikroTik RouterOS add hw-offload management in ip firewall template (#1322)

sjhloco committed:
Fixed missing OK in header of show int ip brief (#1326)

sjhloco committed:
Fixed by adding support for ext comm in asa show bgp summary (#1325)

guillaume-mbali committed:
Fix Cisco: change the ip nat translations template to make ports non-optional (#1329)

whitej6 committed:
resolves inconsistent leading whitespace issue (#1331)

whitej6 committed:
preparing for release (#1332)

jvanderaa committed:
Add dependabot settings. (#1335)

* Add dependabot settings.

guillaume-mbali committed:
Cisco fix access list template

elavaud committed:
Fix huawei display acl all - port numbers replaced by names and IPV6 ACL name position (#1222)

* Fix huawei display acl all - port numbers replaced by names
* Fix Huawei - display acl (ipv6)? all - if name provided, order in IPv6 changes
* Huawei - display acl all - add rule description
* Huawei - display acl all - separately record ACLs and their rules

authored-by: Edouard Lavaud <e.lavaud@unyc.io>

dainok committed:
Fix thp_comware_display_interfacewith productions output (#1338)

elavaud reviewed (approved):
Ok for me but let's see, they might tell you no because it is a breaking change

dainok committed:
Fix hp_comware_display_vlan_all with production outputs (#1339)

dainok committed:
Fix hp_comware_display_lldp_neighbor-information_list with production… (#1340)

* Fix hp_comware_display_lldp_neighbor-information_list with production outputs

* cosmetic

dainok committed:
Fix hp_comware_display_ip_interface with production outputs (#1341)

* Fix hp_comware_display_ip_interface with production outputs

k-ribot committed:
Ubiquiti EdgeRouter corrected IPv6 neighbors template without mac address (#1328)

elavaud committed:
Mikrotik routeros - ip firewall address-list print terse (#1160)

jvanderaa committed:
Moves dependabot to match requirements. (#1345)

k-ribot committed:
Ubiquiti EdgeRouter corrected ARP with incomplete mac address (#1327)

nsnelson402 committed:
Fix for cert output to catch multiword values. (#1316)

Commit:
Re-commit anonymized test data

guillaume-mbali commented:
https://github.com/networktocode/ntc-templates/pull/1353

ReK42 committed:
Updated support for Cisco NXOS devices (#1175)

* Add support for four nxos commands under show ip pim

---------

Co-authored-by: Patryk Szulczewski <patryk@szulczewski.eu>

vladimirs-git committed:
[fix] cisco_nxos_show_interface_brief.textfsm (#1356)

- support interface "Tunnel"
- support sub-interface "Eth1/52.111"
- support mode "f-path" and any others
- support interfaces without ip address, like "--"

diepes committed:
fixs_1060_cisco_ios_show_access-list (#1369)

Co-authored-by: Pieter E Smit <diepes@github.com>

dainok committed:
Support lowercase interface (vasileft) (#1361)

qduk committed:
Update cisco_ios_show_lldp_neighbors_detail parser to gather more information (#1362)

* Updated lldp neighbors detail

* Put yml formatting back

---------

Co-authored-by: Adam Byczkowski <adam.byczkowski@kochgs.com>

diepes committed:
development_scripts sort yaml (#1367)

Co-authored-by: Pieter E Smit <diepes@github.com>

guillaume-mbali committed:
Fix Cisco show interfaces speed and duplex (#1247)

guillaume-mbali committed:
feat(cisco): add vrf template detail (#1372)

diepes committed:
Fixs 1363 arista eos show interfaces - description quotes (#1377)

* fixes_1363 arista eos show interfaces `description could have quotes`

diepes committed:
Fixes#1048 cisco xr admin show inventory (#1118)

* fixes#1048-cisco_xr_admin_show_inventory

---------

Co-authored-by: Pieter E Smit <diepes@github.com>

diepes committed:
Fix1198 cisco nxos show access lists (#1379)

* fix1198 sort existing tests/cisco_nxos/show_access-lists/cisco_nxos_show_access-lists.yml

alicegenkin committed:
Template and tests for cisco_nxos_show_ip_msdp_summary_vrf_all (#1368)

* add templates and tests for cisco_nxos_show_ip_msdp_summary_vrf_all

jvanderaa committed:
Release 3.4.0 (#1378)

sebastiengovh committed:
Adding new outputs for cisco asa show license all command (#1375)

* Adding new outputs for cisco asa show license all command

* fix empty lines in yaml files

tom0010 committed:
Fix #1382 (#1383)

* Fix AS Path regex to account for AS sets

* Update tests for show ip bgp for AS sets

---------

Co-authored-by: tom0010 <tom0010@github.com>

afly007 committed:
Adding Aruba CX template and tests for show system (#1384)

k-ribot committed:
Ubiquiti EdgeRouter corrected show interfaces template (#1355)

SaschaSchwarzK committed:
Added new values for destination port lt/gt/range, IPv6, time-range o… (#1351)

guillaume-mbali committed:
Ubiquiti add nat rules template (#1352)

joewesch committed:
Fixes and Updates CI tests (#1388)

* Renames file to fix test

* Updates Github Actions dependencies

tom0010 committed:
Fix #1386 (#1387)

* Fix NEIGH_AS to support AS dot notation

joewesch committed:
Release v3.5.0 (#1389)

elavaud committed:
feat(cisco_xr): Add VLAN ID to show interfaces template (#1371)

jvanderaa committed:
Updates show transceivers (#1365)

tom0010 committed:
Fix #1390 (#1391)

Change description regex to support whitespaces in description field

mjbear committed:
Change: add tag and id to capture groups for Paloalto interface logical template (#1394)

Modify Palo Alto 'show interface logical' template to also pull id and tag values

tom0010 committed:
FR: Add support for Cisco nxos show bgp vrf all ipv4 unicast neighbors (#1401)

* Add cisco nxos show bgp vrf all ipv4 unicast neighbors template

* Add tests for cisco nxos show bgp vrf all ipv4 unicast neighbors

* Update index for cisco nxos show bgp vrf all ipv4 unicast neighbors

---------

Co-authored-by: tom0010 <tom0010@github.com>

tom0010 committed:
Fix #1321 - Cisco NXOS show ip bgp add support for empty status (#1404)

* Add support for empty status

* Update tests for Cisco NXOS show ip bgp empty status

* Update match compared to value from comments from Ken

* Update tests for show ip bgp nxos

---------

Co-authored-by: tom0010 <tom0010@github.com>

tom0010 committed:
Update cisco nxos show ip bgp neighbors (#1403)

Update Cisco NXOS show ip bgp neighbors

stesla committed:
add support for bgp summary found on 7124SX (#1410)

The output in the raw sample here was collected from an Arista
DCS-7124SX-R switch running EOS version 4.9.5.2.

mjbear committed:
Docs: add invoke local usage example to docs (#1395)

Update invoke usage examples

mjbear committed:
Standardize Mikrotik capture groups to uppercase (#1398)

Changed Mikrotik template capture groups to uppercase per project guidelines

mjbear committed:
Standardize description capture group name (#1399)

* change abbreviation DESC to full word DESCRIPTION to standardize capture
group name

mjbear committed:
Standardize VLAN-related capture group names in the Avaya ERS show vlan template (#1400)

* separate the prefix VLAN and the other words with an underscore to align
with standard conventions

mjbear committed:
Standardize capture group for vlan id (#1416)

* Update index

* separate the prefix VLAN and the other words with an underscore to align
with standard conventions

* standardize [vlan] tag to vlan_id for Palo Alto `show interface logical` (enhance PR 1394)

* replace instances of VLAN with VLAN_ID to standardize/normalize

* update show_macs.yml mock result for the vlan_id capture group change

* fix errant pattern replace
